pkgs/applications/misc/truecrypt/default.nix

```diff
@@ -41,7 +41,7 @@ library, use the 'NOGUI' parameter:
 */
 
 { fetchurl, stdenv, pkgconfig, nasm, fuse, wxGTK, devicemapper,
-  wxGUI ? true
+  wxGUI ? true, gcc
```

Comment:
It is probably better to apply this patch instead: https://git.archlinux.org/svntogit/packages.git/tree/trunk/gcc6.patch?h=packages/truecrypt